src/Jackett.Common/Indexers/ApiBay.cs

```diff
@@ -0,0 +1,250 @@
+            AddCategoryMapping(403, TorznabCatType.ConsolePS4, "PSx");
+            AddCategoryMapping(404, TorznabCatType.ConsoleXbox, "XBOX360");
+            AddCategoryMapping(405, TorznabCatType.ConsoleWii, "Wii");
+            AddCategoryMapping(406, TorznabCatType.ConsoleOther, "Handheld");
+            AddCategoryMapping(407, TorznabCatType.ConsoleOther, "IOS (iPad/iPhone)");
+            AddCategoryMapping(408, TorznabCatType.ConsoleOther, "Android");
+            AddCategoryMapping(499, TorznabCatType.ConsoleOther, "Games Other");
+            // Porn
+            AddCategoryMapping(500, TorznabCatType.XXX, "Porn");
+            AddCategoryMapping(501, TorznabCatType.XXX, "Movies");
+            AddCategoryMapping(502, TorznabCatType.XXXDVD, "Movies DVDR");
+            AddCategoryMapping(503, TorznabCatType.XXXImageset, "Pictures");
+            AddCategoryMapping(504, TorznabCatType.XXX, "Games");
+            AddCategoryMapping(505, TorznabCatType.XXX, "HD - Movies");
+            AddCategoryMapping(506, TorznabCatType.XXX, "Movie Clips");
+            AddCategoryMapping(599, TorznabCatType.XXXOther, "Porn other");
+            // Other
+            AddCategoryMapping(600, TorznabCatType.Other, "Other");
+            AddCategoryMapping(601, TorznabCatType.Books, "E-books");
+            AddCategoryMapping(602, TorznabCatType.BooksComics, "Comics");
+            AddCategoryMapping(603, TorznabCatType.Books, "Pictures");
+            AddCategoryMapping(604, TorznabCatType.Books, "Covers");
+            AddCategoryMapping(605, TorznabCatType.Books, "Physibles");
+            AddCategoryMapping(699, TorznabCatType.BooksOther, "Other Other");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var releases = await PerformQuery(new TorznabQuery());
+
+            await ConfigureIfOK(
+                string.Empty,
+                releases.Any(),
+                () => throw new Exception("Could not find releases from this URL")
+                );
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            // Keywordless search terms return recent torrents rather than no results.
+            if (string.IsNullOrEmpty(query.SearchTerm))
+            {
+                return await GetRecentTorrents();
+            }
+
+            var categories = MapTorznabCapsToTrackers(query);
+
+            var queryStringCategories = string.Join(
+                ",",
+                categories.Count == 0
+                    ? GetAllTrackerCategories()
+                    : categories
+                );
+
+            var response = await RequestWithCookiesAsync(
+                $"{SiteLink}q.php?q={query.SearchTerm}&cat={queryStringCategories}"
```

Comment:
Use `NameValueCollection` object to build the URL for you. It will use URL encode for the parameters and the code is more readable. => https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Indexers/YTS.cs#L101

Reply:
Thank you, just pushed up this change.